### PR TITLE
Improve BankSignering tracking

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Add a new DD action to track which users are elligible for BankSignering when they add to cart

- Centralize all BankSignering events in a single BankSignering service file

## Justify why they are needed

- Improve options for tracking BankSignering conversion funnel

- Centralize all events for it's easy to get an overview

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3080** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3080" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3080?utm_source=stack-comment).

File: apps/store/src/components/ConfirmationPage/ContractCard.tsx [hunk 1]
```diff
@@ -46,7 +47,7 @@ const PendingContractCard = (props: ContractCardProps) => {
   })
 
   const handleClick = (contractId: string) => () => {
-    datadogRum.addAction('BankSignering Initiate', { contractId })
+    datadogRum.addAction(BankSigneringEvent.Initiated, { contractId })
```
Comment (alebedev) [inline apps/store/src/components/ConfirmationPage/ContractCard.tsx:50]: Nit: if we see many events/action that require context, we can add type-safe wrappers like `Tracking` does

**Review by alebedev — APPROVED** [review]
Looks good!

File: apps/store/src/components/ConfirmationPage/ContractCard.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/ConfirmationPage/ContractCard.tsx:50]: yep I was thinking about that -- let's consider it for sure

**Pull request merged**